Src/IronPython/Runtime/Operations/PythonOps.cs

```diff
@@ -1571,7 +1571,9 @@ public static string Repr(CodeContext/*!*/ context, object? o)
                             new ObjectAttributesAdapter(parentContext, backing)
                     };
                     attrdict = new PythonDictionary(new RuntimeVariablesDictionaryStorage(parentStorage, attrStorage));
+#if DEBUG
```

Comment:
I think the Debug.Assert is already `DEBUG` conditional so the `#if DEBUG` is redundant? (Unless I'm confused and it doesn't compile.)

Reply:
I thought so too but it didn't compile. That's why I had to force-push a correction.